src/main/scala/com/netflix/edda/web/FieldSelectorParser.scala

```diff
@@ -164,7 +163,7 @@ class FieldSelectorParser extends RegexParsers {
 
   def invRegexExpr = "!~" ~> regexLiteral ^^ (value => RegexExpr(value, invert = true))
 
-  def id = regex("[a-zA-Z0-9_\\.\\-]*".r)
+  def id = regex("[a-zA-Z0-9_\\.\\-]*[\\s]*[a-zA-Z0-9_\\.\\-]*".r)
```

Comment:
This will only match spaces in the middle, e.g.:

```
scala> Pattern.compile("[a-zA-Z0-9_\\.\\-]*[\\s]*[a-zA-Z0-9_\\.\\-]*")
res0: java.util.regex.Pattern = [a-zA-Z0-9_\.\-]*[\s]*[a-zA-Z0-9_\.\-]*

scala> res0.matcher("a b c").matches
res1: Boolean = false

scala> res0.matcher("a b").matches
res2: Boolean = true
```

It seems like the intent would be:

```
scala> Pattern.compile("[a-zA-Z0-9_\\.\\-\\s]*")
res3: java.util.regex.Pattern = [a-zA-Z0-9_\.\-\s]*

scala> res3.matcher("a b c").matches
res4: Boolean = true

scala> res3.matcher("a b").matches
res5: Boolean = true
```